### PR TITLE
HTML IAM Video Example

### PR DESCRIPTION
**Pull request by davidbielik**
simplified the HTML video docs and added warning for iOS

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/in-app_messages/customize.md [hunk 1]
```diff
@@ -244,56 +244,26 @@ You can also [duplicate]({{ site.baseurl }}/user_guide/engagement_tools/template
...
 
-You are able to display HTML5 videos in our customizable in-app message types with sound included. You can either use an embedded link from a third party (like [Youtube](https://support.google.com/youtube/answer/171780?hl=en)) using a custom HTML5 file or upload a video to your Braze account in the `assets.zip` folder. This uploaded video will then be sent to the device for local playback, so there is no need for a network connection to play the video. This second option is recommended mostly for shorter videos.
+To play a video in an HTML In-App Message, include the following `<video>` element in your HTML, and replace the video names with your file's name (or the remote asset's URL). 
 
-{% tabs %}
-  {% tab Android %}
+To use a local video asset, be sure to include this file when uploading assets to your campaign.
 
-__Android__
+To support iOS devices, you must include the `playsinline` attribute since full screen playback is not support at this time.
```
Comment (Bucimis) [inline _docs/_user_guide/message_building_by_channel/in-app_messages/customize.md]: supported

File: _docs/_user_guide/message_building_by_channel/in-app_messages/customize.md [hunk 2]
```diff
@@ -244,56 +244,26 @@ You can also [duplicate]({{ site.baseurl }}/user_guide/engagement_tools/template
...
-
-Then, copy and paste the following `HTML` snippet into your code, replacing the source parameters with your actual file names.
+Other `<video>` options that can be used can be found [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video).
 
 ```
 <video class="video" autoplay muted playsinline controls>
-  <source src="mov_bbb.mp4" type="video/mp4">
-  <source src="mov_bbb.ogg" type="video/ogg">
-  Your device does not support HTML5 video.
+  <source src="YOUR_VIDEO_FILE.mp4" type="video/mp4">
```
Comment (Bucimis) [inline _docs/_user_guide/message_building_by_channel/in-app_messages/customize.md]: may be good to have this look like a remote url

File: _docs/_user_guide/message_building_by_channel/in-app_messages/customize.md [hunk 3]
```diff
@@ -244,56 +244,26 @@ You can also [duplicate]({{ site.baseurl }}/user_guide/engagement_tools/template
...
+  <source src="YOUR_VIDEO_FILE.ogg" type="video/ogg">
+  Your device does not support playing this video.
```
Comment (Bucimis) [inline _docs/_user_guide/message_building_by_channel/in-app_messages/customize.md]: why is this note needed?

File: _docs/_user_guide/message_building_by_channel/in-app_messages/customize.md [hunk 4]
```diff
@@ -244,56 +244,26 @@ You can also [duplicate]({{ site.baseurl }}/user_guide/engagement_tools/template
...
-
-To use a video in Web in-app messages, embed a link from a third party (like [Youtube](https://support.google.com/youtube/answer/171780?hl=en)) into your [Custom Web Message]({{ site.baseurl }}/user_guide/message_building_by_channel/in-app_messages/customize/#custom-web-message).
+{% alert warning %}
+Full screen videos will not render correctly on iOS and are not supported at this time. You must include the `playsinline` attribute to show the video within the HTML message instead.
+{% endalert %}
 
-{% endtab %}
-{% endtabs %}
+{% alert info %}
+iOS does not support autoplay by default. To update this default option, you can modify the [`ABKInAppMessageHTMLViewController`](https://github.com/Appboy/appboy-ios-sdk/blob/master/AppboyUI/ABKInAppMessage/ViewControllers/ABKInAppMessageHTMLViewController.m)
```
Comment (Bucimis) [inline _docs/_user_guide/message_building_by_channel/in-app_messages/customize.md]: doesn't enabling autoplay currently cause issues? may be better to say it needs to be off

**Comment by KellieHawks** [comment]
@cla-bot check

File: _docs/_user_guide/message_building_by_channel/in-app_messages/customize.md (hunk 4 above)
Comment (davidbielik) [reply _docs/_user_guide/message_building_by_channel/in-app_messages/customize.md]: I'll confirm and update in a future PR if this is wrong.

**Comment by davidbielik** [comment]
this is ready to merge unless anyone has other comments :)

**Comment by KellieHawks** [comment]
> this is ready to merge unless anyone has other comments :)

Merging, thanks!

**Pull request merged**